tests/core/fs/conftest.py

```diff
@@ -202,42 +199,45 @@ def run_as_test(cls):
         pytest.skip()
 
     if param:
+        from parsec.core.fs.storage import UserStorage as RSUserStorage
         from parsec.core.fs.storage import WorkspaceStorage as RSWorkspaceStorage
-        from parsec.core.fs.storage import (
-            workspace_storage_non_speculative_init as rs_workspace_init,
-        )
+        from parsec.core.fs.storage import user_storage_non_speculative_init as rs_user_storage_init
+        from parsec.core.fs.storage import workspace_storage_non_speculative_init as rs_wk_init
         from parsec.core.fs.storage.workspace_storage import (
             DEFAULT_WORKSPACE_STORAGE_CACHE_SIZE,
             FAILSAFE_PATTERN_FILTER,
         )
-        from tests.core.fs.old_workspace_storage import WorkspaceStorage as PyWorkspaceStorage
-        from tests.core.fs.old_workspace_storage import (
-            workspace_storage_non_speculative_init as py_workspace_init,
+        from tests.core.fs.old_storage.user_storage import UserStorage as PyUserStorage
+        from tests.core.fs.old_storage.user_storage import (
+            user_storage_non_speculative_init as py_user_storage_init,
+        )
+        from tests.core.fs.old_storage.workspace_storage import (
+            WorkspaceStorage as PyWorkspaceStorage,
+        )
+        from tests.core.fs.old_storage.workspace_storage import (
+            workspace_storage_non_speculative_init as py_wk_init,
         )
 
-        use_original_wk_storage = True
+        use_rust_impl = True
```

Comment:
`user_fs_use_rust_impl` ? (`use_rust_impl` is to vague of a name, should be rename to something more explicit)

Reply:
I renamed the variable to `use_rust_storage_impl` since it's for patching `WorkspaceStorage` & `UserStorage`